.github/workflows/CI_Pipeline.yml

```diff
@@ -25,19 +25,20 @@ jobs:
     - uses: actions/checkout@v2
     - run: docker build --target test --tag my-test-image .
     - run: docker run --env-file .env.test  my-test-image tests
-    - run: docker run -e MONGO_DB_NAME=e2e_db -e MONGO_CONNECT=${{secrets.MONGO_CONNECT}} -e MONGO_COLLECTION_NAME=${{secrets.MONGO_COLLECTION_NAME}} -e MONGO_DEFAULT_STATUS=${{secrets.MONGO_DEFAULT_STATUS}} my-test-image tests_e2e
-  deploy:
-    name: Deploy build package to Heroku
-    needs: build
-    if: ${{ github.event_name ==  'push' }}
-    runs-on: ubuntu-latest
-    steps:
-    - uses: actions/checkout@v2 # Adds a step to checkout the repository code
+    - run: docker run -e SECRET_KEY=secret -e MONGO_DB_NAME=e2e_db -e MONGO_CONNECT=${{secrets.MONGO_CONNECT}} -e MONGO_COLLECTION_NAME=${{secrets.MONGO_COLLECTION_NAME}} -e MONGO_DEFAULT_STATUS=${{secrets.MONGO_DEFAULT_STATUS}} my-test-image tests_e2e
+##Commenting out as still causing issues
```

Comment:
I don't think you need to comment out the below. When did it cause issues? The only issue before was with e2e tests not the deployment to Heroku.

If you are having issues with it, share the error over Slack and we can advise you.